UnitySDK/Assets/ML-Agents/Scripts/SideChannel/RawBytesChannel.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+namespace MLAgents
+{
+    public class RawBytesChannel : SideChannel
+    {
+
+        private List<byte[]> m_MessagesReceived = new List<byte[]>();
+        private int m_ChannelId;
+
+
+        public RawBytesChannel(int channelId = 0)
+        {
+            m_ChannelId = channelId;
+        }
+        public override int ChannelType()
+        {
+            return (int)SideChannelType.RawBytesChannelStart + m_ChannelId;
+        }
+
+        public override void OnMessageReceived(byte[] data)
+        {
+            m_MessagesReceived.Add(data);
+        }
+
+        public void SendRawBytes(byte[] data)
+        {
+            QueueMessageToSend(data);
+        }
+
+        public IList<byte[]> ReceiveRawBytes()
```

Comment:
`GetReceivedMessages()`? `GetAndClearReceivedMessages()`? "Receive" feels like the wrong verb here.